tests/tests_integration/test_api/test_data_modeling/test_data_models.py

```diff
@@ -115,6 +123,14 @@ def test_retrieve_multiple(self, cognite_client: CogniteClient, cdf_data_models:
         # Assert
         assert [dm.as_id() for dm in retrieved] == ids
 
+    def test_retrieve_with_inline_views(self, cognite_client: CogniteClient, movie_model: DataModel) -> None:
+        # Act
+        retrieved = cognite_client.data_modeling.data_models.retrieve(movie_model.as_id(), inline_views=True)[0]
+
+        # Assert
+        assert retrieved.views is not None
```

Comment:
Let's make DataModel.views a `List ` instead of `Optional[List]` so this check isn't necessary.

Reply:
I agree this is better, but just for my understanding we are breaking the spec (or the spec is documented wrongly) as views are not listed as required.

Reply:
Still think you need to handle that. So Optional[List] in the parameters and then
```
self.views: List[T_View] = views or []
```
in the constructor body.